EntityFrameworkCore.Manipulation.Extensions.IntegrationTests/UpdateTests.cs

```diff
@@ -228,6 +228,51 @@ public async Task UpdateAsync_ShouldReturnCollectionWithOnlyIncludedPropertiesUp
 			context.TestEntitiesWithCompositeKey.Should().BeEquivalentTo(new[] { existingEntities[0], existingEntities[1], expectedUpdatedEntity });
 		}
 
+		[TestMethod]
+		public async Task UpdateAsync_ShouldReturnAffectedUpdatedCollection_WhenASubsetOfEntitiesAreMatchingWithConditionUsingTvpInterceptor()
+		{
+			var existingEntities = Enumerable.Range(0, 52).Select(id => new TestEntityCompositeKey
+			{
+				IdPartA = id.ToString(),
+				IdPartB = "B",
+				IntTestValue = id % 2,
+				BoolTestValue = false,
+				StringTestValue = "short string",
+			}).ToArray();
+			var expectedEntities = Enumerable.Range(0, 52).Select(id => new TestEntityCompositeKey
+			{
+				IdPartA = id.ToString(),
+				IdPartB = "B",
+				IntTestValue = 1,
+				BoolTestValue = true,
+				StringTestValue = "a really long string which is longer than the limit we have on the property",
```

Comment:
I'm not following here. How is this testing that the interceptor gets called?

One test that would be good to have is for the interceptor to store which columns were sent to it, and validate they're the right ones.